src/man/sss_ssh_authorizedkeys.1.xml

```diff
@@ -66,7 +66,7 @@
         <para>
             In addition to the public SSH keys for user
             <replaceable>USER</replaceable>
-            <command>sss_ssh_authorizedkeys</command> can return public SSH keys
+            <command> sss_ssh_authorizedkeys</command> can return public SSH keys
```

Comment:
This should not be needed. It renders correctly on my machine.

Reply:
Hi, @pbrezina

Thank you for your comment.

When I build on my machine, it doesn't seem to insert whitespace.
Therefore, I requested a fix to insert whitespace.

If the result is a problem in my environment, I don't need a patch to insert whitespace.
I would request a PR that removed this patch.

Here is the result of building on my machine:

    # man -l sss_ssh_authorizedkeys.1
    <snip>
    DESCRIPTION
    <snip>
       KEYS FROM CERTIFICATES
           In addition to the public SSH keys for user USERsss_ssh_authorizedkeys can return public SSH keys derived from the public key of a X.509 certificate as well.
                                                       ^^^^^^^^^^^^^^^^^^^^^^^^^^(*)

    # man -l sssd-ldap.5
    <snip>
    DESCRIPTION
    <snip>
           sssddoes not support authentication over an unencrypted channel.
           ^^^^^^^^(*)

Reply:
I sent a PR that removed the patches that added whitespace.